fe/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -554,6 +554,13 @@ public static Catalog getCurrentCatalog() {
         }
     }
 
+    // NOTICE: In most cases, you should use getCurrentCatalog() to get the catalog.
+    // getServiingCatalog() is only used for Checkpoint thread and some special places
```

Comment:
getServingCatalog

Reply:
Oh, I moved the change to PR #3229
I will rebase after that PR merged.

Reply:
Already for review.